src/Presentation/RestApi/Controller/v1/SdkInitProjectController.php

```diff
@@ -7,19 +7,32 @@
 
 namespace SprykerSdk\Sdk\Presentation\RestApi\Controller\v1;
 
-use SprykerSdk\Sdk\Presentation\RestApi\Enum\OpenApiType;
+use SprykerSdk\Sdk\Presentation\RestApi\Processor\SdkInitProjectProcessor;
 use Symfony\Component\HttpFoundation\JsonResponse;
 use Symfony\Component\HttpFoundation\Request;
 
 class SdkInitProjectController extends BaseController
 {
+    /**
```

Comment:
I see improvements for this controller, but there are no improvements/changes provided in tests. Why?

Reply:
added acceptance test